eventtracking/django/tests/test_models.py

```diff
@@ -0,0 +1,113 @@
+
+        first_enabled = []
+        first_disabled = []
+        second_enabled = []
+        second_disabled = []
+
+        for _ in range(5):
+            first_enabled.append(RegExFilterFactory(is_enabled=True, backend_name='first'))
+            first_disabled.append(RegExFilterFactory(is_enabled=False, backend_name='first'))
+            second_enabled.append(RegExFilterFactory(is_enabled=True, backend_name='second'))
+            second_disabled.append(RegExFilterFactory(is_enabled=False, backend_name='second'))
+
+        # test with no backend name
+        self.assertEqual(second_enabled[4], RegExFilter.get_latest_enabled_filter())
+
+        # test with backend backend name
+        self.assertEqual(first_enabled[4], RegExFilter.get_latest_enabled_filter('first'))
+
+        # test enabling a filter
+        first_disabled[2].is_enabled = True
+        first_disabled[2].save()
+        self.assertEqual(first_disabled[2], RegExFilter.get_latest_enabled_filter())
+        self.assertEqual(first_disabled[2], RegExFilter.get_latest_enabled_filter('first'))
+        self.assertEqual(second_enabled[4], RegExFilter.get_latest_enabled_filter('second'))
+
+        # test modifying an enabled filter
+        second_enabled[1].save()
+        self.assertEqual(second_enabled[1], RegExFilter.get_latest_enabled_filter())
+        self.assertEqual(first_disabled[2], RegExFilter.get_latest_enabled_filter('first'))
+        self.assertEqual(second_enabled[1], RegExFilter.get_latest_enabled_filter('second'))
+
+    def test_validator(self):
+        valid_expressions = ['abc', '.?[1-10]*']
+        invalid_expressions = ['***', '*?*']
+        expressions_string = ' \n '.join(valid_expressions+invalid_expressions)
+
+        expected_error_message = 'The following invalid expressions must be corrected: {}'.format(
+            ', '.join(invalid_expressions)
+        )
+
+        with self.assertRaisesMessage(ValidationError, expected_error_message):
+            validate_regex_list(expressions_string)
+
+    @patch('eventtracking.django.models.logger')
+    def test_with_invalid_regular_expressions(self, mocked_logger):
+        invalid_regular_exressions = ['***', '*?*']
+        valid_regular_expressions = ['abc', '.?[1-10]*']
+
+        expressions_string = ' \n '.join(invalid_regular_exressions + valid_regular_expressions)
+        regex_filter = RegExFilterFactory(
+            regular_expressions=expressions_string
+        )
+
+        compiled_regular_expressions = regex_filter.compiled_expressions
+        self.assertEqual(compiled_regular_expressions, [
+            re.compile(ex) for ex in valid_regular_expressions
+        ])
+        mocked_logger.error.assert_called_once_with(
+            'The following invalid expressions were found in the filter: %s',
+            ', '.join(invalid_regular_exressions)
+        )
+
+    @ddt.data(
+        ('whitelist', 'sentinel.* \n not_matching \n .*', True),
```

Comment:
We are trying to change this language across our code base from whitelist/blacklist to allowlist/blocklist. Could we make that change here as well?